app/src/Toolbox.tsx

```diff
@@ -37,10 +37,10 @@ export function Toolbox({
           <option>Lottery</option>
           <option>Governance</option>
           <option disabled>Auction</option>
-          <option disabled>MarketPlace</option>
+          <option>MarketPlace</option>
           <option disabled>FundMe</option>
           <option disabled>Math</option>
```

Comment:
```suggestion
```
Can we just remove these that we wont create?

Reply:
Removed